pkg/plugins/golang/v3/api.go

```diff
@@ -189,7 +190,7 @@ func (p *createAPISubcommand) Validate() error {
 		}
 
 		// Check CRDVersion against all other CRDVersions in p.config for compatibility.
-		if !p.config.IsCRDVersionCompatible(p.resource.API.CRDVersion) {
+		if !pluginutil.IsOnlyCRDVersion(p.config, p.resource.API.CRDVersion) {
```

Comment:
I do not think that this change is nice. I did not like the idea of have this funcs IsOnlyCRDVersion. The previous one represents a better need. We will pass the version of CRD or Webhook which we want to use and check if the project config allows or not it.